src/Symfony/Component/Validator/Tests/Constraints/EmailValidatorTest.php

```diff
@@ -125,4 +125,19 @@ public function getDnsChecks()
             array('AAAA', true),
         );
     }
+
+    public function testHostnameIsProperlyParsed()
+    {
+        DnsMock::withMockedHosts(array(
+            'baz.com' => array(array('type' => 'MX')),
+            '@bar"@baz.com' => array(array('type' => false)),
```

Comment:
you don't need this line: the list of moked hosts is exclusive and a look for this would fail anyway